packages/app-page-builder/src/blockEditor/config/editorBar/SaveBlockButton/SaveBlockButton.tsx

```diff
@@ -19,11 +19,10 @@ const DefaultSaveBlockButton: React.FC = () => {
             new SaveBlockActionEvent({
                 debounce: false,
                 onFinish() {
-                    history.push(`/page-builder/page-blocks`);
-
                     // Let's wait a bit, because we are also redirecting the user.
                     setTimeout(() => {
-                        showSnackbar(`Block ${block.name} saved successfully!`);
+                        showSnackbar(`Block "${block.name}" saved successfully!`);
+                        history.push(`/page-builder/page-blocks`);
                     }, 500);
```

Comment:
Just curious, why do we need a timeout here? This is not a change request, just want to know, because snackbar should be visible even after router change.

Reply:
@neatbyte-vnobis To be honest, I think this is a leftover from a long time ago. If you can, please try removing it and see if it works. If not, leave it as is. Thanks a lot!